README.md

```diff
@@ -16,6 +18,8 @@
 
 [Live Demo Link](https://verissimofrancis-air-pollution.netlify.app)
```

Comment:
- [ ] This live demo link is not working as expected. Please update the link.

Reply:
Resolved @mmsesay

Reply:
Thanks for the feedback